common/src/main/resources/runtime.properties

```diff
@@ -37,6 +37,11 @@
 # If true, Falcon skips oozie dryrun while scheduling entities.
 *.falcon.skip.dryrun=false
 
+# Minimum value that the Falcon Oozie Coordinator Builder will use when 
+# configuring the throttle for a coordinator. It is recommended that the
+# minimum be greater than or equal to the Oozie default of 12. 
+*.falcon.oozie.coordinator.throttle.min=12
```

Comment:
Thanks for the feedback. While I was careful to include 'oozie' in the property name, I think it's good to explicitly call it out in the description. I'll make that change.